packages/dx-react-grid/package.json

```diff
@@ -60,29 +62,22 @@
     "enzyme": "3.8.0",
     "enzyme-adapter-react-16": "^1.8.0",
     "enzyme-to-json": "^3.3.5",
-    "eslint": "^5.13.0",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-filenames": "^1.3.2",
-    "eslint-plugin-import": "^2.16.0",
-    "eslint-plugin-jest": "^22.2.2",
-    "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-react": "^7.12.4",
     "jest": "^24.0.0",
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
     "react-test-renderer": "^16.7.0",
     "rollup": "^1.1.2",
     "rollup-plugin-babel": "^4.3.2",
     "rollup-plugin-license": "^0.8.1",
-    "rollup-plugin-node-resolve": "^4.0.0"
-  },
-  "dependencies": {
-    "@devexpress/dx-grid-core": "1.10.1",
-    "prop-types": "^15.6.2"
+    "rollup-plugin-node-resolve": "^4.0.0",
```

Comment:
Should we remove these deps: rollup-plugin-node-resolve, rollup-plugin-babel ?

We don't use them anymore in rollup.config.